src/console/utilities/sequences/tse/tse_3d.py

```diff
@@ -165,6 +190,26 @@ def constructor(
     seq.set_definition("train_duration_tr", train_duration_tr)
     seq.set_definition("tr_delay", tr_delay)
 
-    return (seq, trains)
+    return (seq, pe_traj)
 
-# %%
+
+
+def sort_kspace(kspace: np.ndarray, trajectory: np.ndarray, dim: Dimensions) -> np.ndarray:
+    """
+    Sort acquired k-space lines.
+
+    Parameters
+    ----------
+    kspace
+        Acquired k-space data in the format (averages, coils, pe, ro)
```

Comment:
Although 'pe' and 'ro' are well-known keywords for MRI people, I consider it could be nice to have an abbreviations line in the function description for readibility.

Reply:
You're right, I will update the docstring

Reply:
I will postpone this change since @LUMC-LowFieldMRI has been working on the trajectory of the 3D TSE sequence. We will update the docstring in a separate PR.